packages/qdrant-loader-core/tests/test_graph_store.py

```
@@ -0,0 +1,92 @@
+from qdrant_loader_core.graph.models import GraphNode, GraphEdge, SubGraph
+
+import pytest
+from unittest.mock import AsyncMock
+
+from qdrant_loader_core.graph.store import GraphStore
+
+
+class DummyGraphStore(GraphStore):
+    def __init__(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)